app/README-Google.md

```diff
@@ -0,0 +1,35 @@
+# Configuring Google Sign In
```

Comment:
I'll adopt the title you used in your readme, "Using AppAuth for Android with Google".